.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+"on":
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  test_matrix:
+    strategy:
+      matrix:
+        node-version:
+          - 10
+          - 12
+          - 14
```

Comment:
What does this directive actually do, given that the `package-lock.json` requires `14` or later and the test directive at line 41 appears to specify `16`?  Do lines 15 and 16 have any effect (other than to run extra stuff which is skipped/unworkable/redundant)?

Reply:
Ha!  Nevermind -- this was fixed in #6.  😊